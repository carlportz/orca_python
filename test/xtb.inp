$constrain
    force constant=1.5
    angle: 1, 2, 3, 120.0
$end