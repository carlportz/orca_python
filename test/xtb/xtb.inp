$constrain
    force constant=1.5
    angle: 2, 1, 3, 120.0
$end