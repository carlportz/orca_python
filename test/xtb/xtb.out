      -----------------------------------------------------------      
     |                   =====================                   |     
     |                           x T B                           |     
     |                   =====================                   |     
     |                         S. Grimme                         |     
     |          Mulliken Center for Theoretical Chemistry        |     
     |                    University of Bonn                     |     
      -----------------------------------------------------------      

   * xtb version 6.5.1 (fef0646) compiled by 'conda@b96dff2e9fd6' on 2023-01-04

   xtb is free software: you can redistribute it and/or modify it under
   the terms of the GNU Lesser General Public License as published by
   the Free Software Foundation, either version 3 of the License, or
   (at your option) any later version.
   
   xtb is distributed in the hope that it will be useful,
   but WITHOUT ANY WARRANTY; without even the implied warranty of
   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
   GNU Lesser General Public License for more details.
   
   Cite this work as:
   * C. Bannwarth, E. Caldeweyher, S. Ehlert, A. Hansen, P. Pracht,
     J. Seibert, S. Spicher, S. Grimme, WIREs Comput. Mol. Sci., 2020, 11,
     e01493. DOI: 10.1002/wcms.1493
   
   for GFN2-xTB:
   * C. Bannwarth, S. Ehlert and S. Grimme., J. Chem. Theory Comput., 2019,
     15, 1652-1671. DOI: 10.1021/acs.jctc.8b01176
   for GFN1-xTB:
   * S. Grimme, C. Bannwarth, P. Shushkov, J. Chem. Theory Comput., 2017,
     13, 1989-2009. DOI: 10.1021/acs.jctc.7b00118
   for GFN0-xTB:
   * P. Pracht, E. Caldeweyher, S. Ehlert, S. Grimme, ChemRxiv, 2019, preprint.
     DOI: 10.26434/chemrxiv.8326202.v1
   for GFN-FF:
   * S. Spicher and S. Grimme, Angew. Chem. Int. Ed., 2020, 59, 15665-15673.
     DOI: 10.1002/anie.202004239
   
   for ALPB and GBSA implicit solvation:
   * S. Ehlert, M. Stahn, S. Spicher, S. Grimme, J. Chem. Theory Comput.,
     2021, 17, 4250-4261. DOI: 10.1021/acs.jctc.1c00471
   
   for DFT-D4:
   * E. Caldeweyher, C. Bannwarth and S. Grimme, J. Chem. Phys., 2017,
     147, 034112. DOI: 10.1063/1.4993215
   * E. Caldeweyher, S. Ehlert, A. Hansen, H. Neugebauer, S. Spicher,
     C. Bannwarth and S. Grimme, J. Chem. Phys., 2019, 150, 154122.
     DOI: 10.1063/1.5090222
   * E. Caldeweyher, J.-M. Mewes, S. Ehlert and S. Grimme, Phys. Chem. Chem. Phys.
     2020, 22, 8499-8512. DOI: 10.1039/D0CP00502A
   
   for sTDA-xTB:
   * S. Grimme and C. Bannwarth, J. Chem. Phys., 2016, 145, 054103.
     DOI: 10.1063/1.4959605
   
   in the mass-spec context:
   * V. Asgeirsson, C. Bauer and S. Grimme, Chem. Sci., 2017, 8, 4879.
     DOI: 10.1039/c7sc00601b
   * J. Koopman and S. Grimme, ACS Omega 2019, 4, 12, 15120-15133.
     DOI: 10.1021/acsomega.9b02011
   
   for metadynamics refer to:
   * S. Grimme, J. Chem. Theory Comput., 2019, 155, 2847-2862
     DOI: 10.1021/acs.jctc.9b00143
   
   for SPH calculations refer to:
   * S. Spicher and S. Grimme, J. Chem. Theory Comput., 2021, 17, 1701-1714
     DOI: 10.1021/acs.jctc.0c01306
   
   with help from (in alphabetical order)
   P. Atkinson, C. Bannwarth, F. Bohle, G. Brandenburg, E. Caldeweyher
   M. Checinski, S. Dohm, S. Ehlert, S. Ehrlich, I. Gerasimov, C. Hölzer
   A. Katbashev, J. Koopman, C. Lavigne, S. Lehtola, F. März, M. Müller,
   F. Musil, H. Neugebauer, J. Pisarek, C. Plett, P. Pracht, F. Pultar,
   J. Seibert, P. Shushkov, S. Spicher, M. Stahn, M. Steiner, T. Strunk,
   J. Stückrath, T. Rose, and J. Unsleber
   
 * started run on 2025/04/03 at 11:34:10.703     

           -------------------------------------------------
          |                Calculation Setup                |
           -------------------------------------------------

          program call               : /home/kreimendahl/miniforge3/envs/chem-env/bin/xtb --opt --namespace xtb --gfn 2 --parallel 2 --input /home/kreimendahl/orca_python/test/xtb/xtb.inp -- /home/kreimendahl/orca_python/test/xtb/xtb_input.xyz
          calculation namespace      : xtb
          coordinate file            : /home/kreimendahl/orca_python/test/xtb/xtb_input.xyz
          omp threads                :                     2

   ID    Z sym.   atoms
    1    8 O      1
    2    1 H      2, 3
constraining angle 2 1 3 to  120.0000000°, actual value:  104.5559749°

           -------------------------------------------------
          |                 G F N 2 - x T B                 |
           -------------------------------------------------

        Reference                      10.1021/acs.jctc.8b01176
      * Hamiltonian:
        H0-scaling (s, p, d)           1.850000    2.230000    2.230000
        zeta-weighting                 0.500000
      * Dispersion:
        s8                             2.700000
        a1                             0.520000
        a2                             5.000000
        s9                             5.000000
      * Repulsion:
        kExp                           1.500000    1.000000
        rExp                           1.000000
      * Coulomb:
        alpha                          2.000000
        third order                    shell-resolved
        anisotropic                    true
        a3                             3.000000
        a5                             4.000000
        cn-shift                       1.200000
        cn-exp                         4.000000
        max-rad                        5.000000


          ...................................................
          :                      SETUP                      :
          :.................................................:
          :  # basis functions                   6          :
          :  # atomic orbitals                   6          :
          :  # shells                            4          :
          :  # electrons                         8          :
          :  max. iterations                   250          :
          :  Hamiltonian                  GFN2-xTB          :
          :  restarted?                      false          :
          :  GBSA solvation                  false          :
          :  PC potential                    false          :
          :  electronic temp.          300.0000000     K    :
          :  accuracy                    1.0000000          :
          :  -> integral cutoff          0.2500000E+02      :
          :  -> integral neglect         0.1000000E-07      :
          :  -> SCF convergence          0.1000000E-05 Eh   :
          :  -> wf. convergence          0.1000000E-03 e    :
          :  Broyden damping             0.4000000          :
          ...................................................

 iter      E             dE          RMSdq      gap      omega  full diag
   1     -5.1044076 -0.510441E+01  0.430E+00   15.08       0.0  T
   2     -5.1056163 -0.120874E-02  0.250E+00   14.81       1.0  T
   3     -5.1058430 -0.226703E-03  0.376E-01   14.59       1.0  T
   4     -5.1059049 -0.618589E-04  0.857E-02   14.74       1.0  T
   5     -5.1059073 -0.247724E-05  0.528E-02   14.69       1.1  T
   6     -5.1059089 -0.159142E-05  0.125E-03   14.71      46.3  T
   7     -5.1059089 -0.456751E-09  0.941E-04   14.71      61.4  T
   8     -5.1059089 -0.519973E-09  0.173E-05   14.71    3334.1  T

   *** convergence criteria satisfied after 8 iterations ***

         #    Occupation            Energy/Eh            Energy/eV
      -------------------------------------------------------------
         1        2.0000           -0.6819145             -18.5558
         2        2.0000           -0.5666953             -15.4206
         3        2.0000           -0.5146536             -14.0044
         4        2.0000           -0.4475638             -12.1788 (HOMO)
         5                          0.0930594               2.5323 (LUMO)
         6                          0.2497655               6.7965
      -------------------------------------------------------------
                  HL-Gap            0.5406231 Eh           14.7111 eV
             Fermi-level           -0.1772522 Eh           -4.8233 eV

 SCC (total)                   0 d,  0 h,  0 min,  0.002 sec
 SCC setup                      ...        0 min,  0.000 sec (  8.360%)
 Dispersion                     ...        0 min,  0.000 sec (  1.522%)
 classical contributions        ...        0 min,  0.000 sec (  3.553%)
 integral evaluation            ...        0 min,  0.000 sec (  7.208%)
 iterations                     ...        0 min,  0.001 sec ( 63.739%)
 molecular gradient             ...        0 min,  0.000 sec ( 10.010%)
 printout                       ...        0 min,  0.000 sec (  4.551%)

         :::::::::::::::::::::::::::::::::::::::::::::::::::::
         ::                     SUMMARY                     ::
         :::::::::::::::::::::::::::::::::::::::::::::::::::::
         :: total energy              -4.961366917990 Eh    ::
         :: gradient norm              0.966507928285 Eh/a0 ::
         :: HOMO-LUMO gap             14.711104682377 eV    ::
         ::.................................................::
         :: SCC energy                -5.105908931635 Eh    ::
         :: -> isotropic ES            0.031402487387 Eh    ::
         :: -> anisotropic ES          0.000582039930 Eh    ::
         :: -> anisotropic XC         -0.000858854792 Eh    ::
         :: -> dispersion             -0.000141165660 Eh    ::
         :: repulsion energy           0.035556948177 Eh    ::
         :: add. restraining           0.108985065438 Eh    ::
         :: total charge               0.000000000000 e     ::
         :::::::::::::::::::::::::::::::::::::::::::::::::::::

      ----------------------------------------------------------- 
     |                   =====================                   |
     |                        A N C O P T                        |
     |                   =====================                   |
     |               Approximate Normal Coordinate               |
     |                Rational Function Optimizer                |
      ----------------------------------------------------------- 

          ...................................................
          :                      SETUP                      :
          :.................................................:
          :   optimization level            normal          :
          :   max. optcycles                   200          :
          :   ANC micro-cycles                  20          :
          :   degrees of freedom                 3          :
          :.................................................:
          :   RF solver                   davidson          :
          :   write xtbopt.log                true          :
          :   linear?                        false          :
          :   energy convergence         0.5000000E-05 Eh   :
          :   grad. convergence          0.1000000E-02 Eh/α :
          :   maximium RF displ.         1.0000000          :
          :   Hlow (freq-cutoff)         0.1000000E-01      :
          :   Hmax (freq-cutoff)         5.0000000          :
          :   S6 in model hess.         20.0000000          :
          ...................................................

generating ANC from model Hessian ...
Using Lindh-Hessian (1995)
 Shifting diagonal of input Hessian by    0.0000000000000000     
 Lowest  eigenvalues of input Hessian
    0.000000    0.000000    0.000000    0.000000    0.000000    0.000000
    0.842649    1.896620    4.367526
 Highest eigenvalues
    0.000000    0.000000    0.000000    0.842649    1.896620    4.367526


........................................................................
.............................. CYCLE    1 ..............................
........................................................................
   1     -5.1059089 -0.510591E+01  0.205E-06   14.71       0.0  T
   2     -5.1059089 -0.266454E-14  0.996E-07   14.71   57982.0  T
   3     -5.1059089 -0.888178E-15  0.330E-07   14.71  100000.0  T
     SCC iter.                  ...        0 min,  0.001 sec
     gradient                   ...        0 min,  0.000 sec
 * total energy  :    -4.9613669 Eh     change       -0.1749711E-12 Eh
   gradient norm :     0.9664619 Eh/α   predicted     0.0000000E+00 (-100.00%)
   displ. norm   :     0.2120025 α      lambda       -0.2045192E+00
   maximum displ.:     0.2113501 α      in ANC's #3, #1, #2, ...

........................................................................
.............................. CYCLE    2 ..............................
........................................................................
   1     -5.1148327 -0.511483E+01  0.704E-01   15.46       0.0  T
   2     -5.1149634 -0.130632E-03  0.371E-01   15.37       1.0  T
   3     -5.1149873 -0.239347E-04  0.996E-02   15.31       1.0  T
   4     -5.1149888 -0.145742E-05  0.118E-02   15.33       4.9  T
   5     -5.1149888 -0.125726E-08  0.331E-03   15.33      17.5  T
   6     -5.1149888 -0.361585E-08  0.128E-04   15.33     450.9  T
   7     -5.1149888 -0.186517E-12  0.296E-05   15.33    1952.7  T
     SCC iter.                  ...        0 min,  0.001 sec
     gradient                   ...        0 min,  0.000 sec
 * total energy  :    -5.0650854 Eh     change       -0.1037185E+00 Eh
   gradient norm :     0.0621897 Eh/α   predicted    -0.1068557E+00 (   3.02%)
   displ. norm   :     0.0707993 α      lambda       -0.4260725E-02
   maximum displ.:     0.0707276 α      in ANC's #1, #2, #3, ...

........................................................................
.............................. CYCLE    3 ..............................
........................................................................
   1     -5.1053655 -0.510537E+01  0.269E-01   13.91       0.0  T
   2     -5.1053751 -0.965566E-05  0.150E-01   13.94       1.0  T
   3     -5.1053768 -0.166464E-05  0.287E-02   13.95       2.0  T
   4     -5.1053770 -0.207718E-06  0.555E-03   13.94      10.4  T
   5     -5.1053770 -0.100392E-08  0.465E-04   13.94     124.2  T
   6     -5.1053770 -0.778213E-10  0.409E-05   13.94    1411.8  T
     SCC iter.                  ...        0 min,  0.000 sec
     gradient                   ...        0 min,  0.000 sec
 * total energy  :    -5.0673444 Eh     change       -0.2258987E-02 Eh
   gradient norm :     0.0272765 Eh/α   predicted    -0.2141041E-02 (  -5.22%)
   displ. norm   :     0.0144102 α      lambda       -0.2536737E-03
   maximum displ.:     0.0126118 α      in ANC's #1, #3, #2, ...

........................................................................
.............................. CYCLE    4 ..............................
........................................................................
   1     -5.1037788 -0.510378E+01  0.337E-02   13.72       0.0  T
   2     -5.1037788 -0.161591E-07  0.200E-02   13.72       2.9  T
   3     -5.1037788 -0.678917E-08  0.123E-03   13.72      46.8  T
   4     -5.1037788 -0.402360E-09  0.420E-04   13.72     137.4  T
   5     -5.1037788 -0.169260E-10  0.175E-05   13.72    3308.5  T
     SCC iter.                  ...        0 min,  0.000 sec
     gradient                   ...        0 min,  0.000 sec
 * total energy  :    -5.0674801 Eh     change       -0.1356587E-03 Eh
   gradient norm :     0.0013573 Eh/α   predicted    -0.1268626E-03 (  -6.48%)
   displ. norm   :     0.0026016 α      lambda       -0.2012462E-05
   maximum displ.:     0.0025176 α      in ANC's #1, #3, #2, ...

........................................................................
.............................. CYCLE    5 ..............................
........................................................................
   1     -5.1034400 -0.510344E+01  0.797E-03   13.67       0.0  T
   2     -5.1034400 -0.489334E-08  0.458E-03   13.67      12.6  T
   3     -5.1034400 -0.930874E-09  0.652E-04   13.67      88.5  T
   4     -5.1034400 -0.140683E-09  0.146E-04   13.67     394.6  T
     SCC iter.                  ...        0 min,  0.000 sec
     gradient                   ...        0 min,  0.000 sec
 * total energy  :    -5.0674810 Eh     change       -0.8644865E-06 Eh
   gradient norm :     0.0007177 Eh/α   predicted    -0.1006656E-05 (  16.45%)
   displ. norm   :     0.0007446 α      lambda       -0.2639044E-06
   maximum displ.:     0.0007067 α      in ANC's #1, #3, #2, ...

   *** GEOMETRY OPTIMIZATION CONVERGED AFTER 5 ITERATIONS ***

------------------------------------------------------------------------
 total energy gain   :        -0.1061140 Eh      -66.5876 kcal/mol
 total RMSD          :         0.1400751 a0        0.0741 Å
 total power (kW/mol):       -55.7204728 (step)********** (real)
------------------------------------------------------------------------

 ANCopt (total)                0 d,  0 h,  0 min,  0.006 sec
 optimizer setup                ...        0 min,  0.000 sec (  1.844%)
 model hessian                  ...        0 min,  0.000 sec (  1.271%)
 ANC generation                 ...        0 min,  0.000 sec (  1.096%)
 coordinate transformation      ...        0 min,  0.000 sec (  0.367%)
 single point calculation       ...        0 min,  0.004 sec ( 66.185%)
 optimization log               ...        0 min,  0.001 sec ( 18.920%)
 hessian update                 ...        0 min,  0.000 sec (  0.497%)
 rational function              ...        0 min,  0.000 sec (  4.063%)

================
 final structure:
================
3
 xtb: 6.5.1 (fef0646)
O            0.05443935805899        0.00000000000000        0.04329509248034
H           -0.06896567556398       -0.00000000000000        0.98912820898008
H            0.93842631750499       -0.00000000000000       -0.31512330146043

 Bond Distances (Angstroems)
 ---------------------------
O1-H2=0.9538         O1-H3=0.9539         H2-O1=0.9538         H3-O1=0.9539         
 O  H  Rav=0.9539 sigma=0.0000  Rmin=0.9538  Rmax=0.9539     2

 selected bond angles (degree)
 --------------------
H3-O1-H2=119.50                

 selected dihedral angles (degree)
 ---------------------------------

           -------------------------------------------------
          |                Final Singlepoint                |
           -------------------------------------------------

          ...................................................
          :                      SETUP                      :
          :.................................................:
          :  # basis functions                   6          :
          :  # atomic orbitals                   6          :
          :  # shells                            4          :
          :  # electrons                         8          :
          :  max. iterations                   250          :
          :  Hamiltonian                  GFN2-xTB          :
          :  restarted?                      false          :
          :  GBSA solvation                  false          :
          :  PC potential                    false          :
          :  electronic temp.          300.0000000     K    :
          :  accuracy                    1.0000000          :
          :  -> integral cutoff          0.2500000E+02      :
          :  -> integral neglect         0.1000000E-07      :
          :  -> SCF convergence          0.1000000E-05 Eh   :
          :  -> wf. convergence          0.1000000E-03 e    :
          :  Broyden damping             0.4000000          :
          ...................................................

 iter      E             dE          RMSdq      gap      omega  full diag
   1     -5.1034400 -0.510344E+01  0.112E-06   13.67       0.0  T
   2     -5.1034400 -0.266454E-14  0.664E-07   13.67   86926.5  T
   3     -5.1034400 -0.355271E-14  0.138E-07   13.67  100000.0  T

   *** convergence criteria satisfied after 3 iterations ***

         #    Occupation            Energy/Eh            Energy/eV
      -------------------------------------------------------------
         1        2.0000           -0.6741889             -18.3456
         2        2.0000           -0.5751873             -15.6516
         3        2.0000           -0.4921552             -13.3922
         4        2.0000           -0.4403301             -11.9820 (HOMO)
         5                          0.0621454               1.6911 (LUMO)
         6                          0.3092527               8.4152
      -------------------------------------------------------------
                  HL-Gap            0.5024755 Eh           13.6731 eV
             Fermi-level           -0.1890924 Eh           -5.1455 eV

 SCC (total)                   0 d,  0 h,  0 min,  0.001 sec
 SCC setup                      ...        0 min,  0.000 sec ( 11.334%)
 Dispersion                     ...        0 min,  0.000 sec (  2.026%)
 classical contributions        ...        0 min,  0.000 sec (  0.964%)
 integral evaluation            ...        0 min,  0.000 sec ( 13.899%)
 iterations                     ...        0 min,  0.000 sec ( 40.990%)
 molecular gradient             ...        0 min,  0.000 sec ( 18.509%)
 printout                       ...        0 min,  0.000 sec (  9.750%)

         :::::::::::::::::::::::::::::::::::::::::::::::::::::
         ::                     SUMMARY                     ::
         :::::::::::::::::::::::::::::::::::::::::::::::::::::
         :: total energy              -5.067480954245 Eh    ::
         :: gradient norm              0.000717354878 Eh/a0 ::
         :: HOMO-LUMO gap             13.673054669995 eV    ::
         ::.................................................::
         :: SCC energy                -5.103440033219 Eh    ::
         :: -> isotropic ES            0.032533346643 Eh    ::
         :: -> anisotropic ES         -0.000470352155 Eh    ::
         :: -> anisotropic XC         -0.001174747967 Eh    ::
         :: -> dispersion             -0.000139648266 Eh    ::
         :: repulsion energy           0.035846642275 Eh    ::
         :: add. restraining           0.000112436705 Eh    ::
         :: total charge              -0.000000000000 e     ::
         :::::::::::::::::::::::::::::::::::::::::::::::::::::

           -------------------------------------------------
          |                Property Printout                |
           -------------------------------------------------

    * Orbital Energies and Occupations

         #    Occupation            Energy/Eh            Energy/eV
      -------------------------------------------------------------
         1        2.0000           -0.6741889             -18.3456
         2        2.0000           -0.5751873             -15.6516
         3        2.0000           -0.4921552             -13.3922
         4        2.0000           -0.4403301             -11.9820 (HOMO)
         5                          0.0621454               1.6911 (LUMO)
         6                          0.3092527               8.4152
      -------------------------------------------------------------
                  HL-Gap            0.5024755 Eh           13.6731 eV
             Fermi-level           -0.1890924 Eh           -5.1455 eV

     #   Z          covCN         q      C6AA      α(0)
     1   8 O        1.612    -0.578    24.675     6.704
     2   1 H        0.806     0.289     0.754     1.364
     3   1 H        0.806     0.289     0.754     1.364

 Mol. C6AA /au·bohr⁶  :         44.625990
 Mol. C8AA /au·bohr⁸  :        798.917431
 Mol. α(0) /au        :          9.432300


Wiberg/Mayer (AO) data.
largest (>0.10) Wiberg bond orders for each atom

 ---------------------------------------------------------------------------
     #   Z sym  total        # sym  WBO       # sym  WBO       # sym  WBO
 ---------------------------------------------------------------------------
     1   8 O    1.829 --     3 H    0.915     2 H    0.915
     2   1 H    0.916 --     1 O    0.915
     3   1 H    0.916 --     1 O    0.915
 ---------------------------------------------------------------------------

Topologies differ in total number of bonds
Writing topology from bond orders to xtbtopo.mol

molecular dipole:
                 x           y           z       tot (Debye)
 q only:        0.416      -0.000       0.321
   full:        0.611       0.000       0.472       1.962
molecular quadrupole (traceless):
                xx          xy          yy          xz          yz          zz
 q only:        0.354      -0.000      -1.009      -0.571       0.000       0.655
  q+dip:        0.450       0.000      -1.332      -0.818      -0.000       0.882
   full:        0.559       0.000      -1.569      -0.856      -0.000       1.010

           -------------------------------------------------
          |                Geometry Summary                 |
           -------------------------------------------------

      molecular mass/u    :       18.0152864
   center of mass at/Å    :        0.0969933      -0.0000000       0.0761605
  moments of inertia/u·Å² :        0.4133557E+00   0.1368739E+01   0.1782094E+01
rotational constants/cm⁻¹ :        0.4078239E+02   0.1231618E+02   0.9459450E+01

 * 2 selected distances

     #   Z          #   Z                                           value/Å
     1   8 O        2   1 H                                       0.9538497 (min)
     1   8 O        3   1 H                                       0.9538851 (max)

 * 1 distinct bonds (by element types)

   Z      Z             #   av. dist./Å        max./Å        min./Å
   1 H    8 O           2     0.9538674     0.9538851     0.9538497


optimized geometry written to: xtbopt.xyz


           -------------------------------------------------
          | TOTAL ENERGY               -5.067480954245 Eh   |
          | GRADIENT NORM               0.000717354878 Eh/α |
          | HOMO-LUMO GAP              13.673054669995 eV   |
           -------------------------------------------------

------------------------------------------------------------------------
 * finished run on 2025/04/03 at 11:34:10.762     
------------------------------------------------------------------------
 total:
 * wall-time:     0 d,  0 h,  0 min,  0.059 sec
 *  cpu-time:     0 d,  0 h,  0 min,  0.324 sec
 * ratio c/w:     5.511 speedup
 SCF:
 * wall-time:     0 d,  0 h,  0 min,  0.002 sec
 *  cpu-time:     0 d,  0 h,  0 min,  0.000 sec
 * ratio c/w:     0.000 speedup
 ANC optimizer:
 * wall-time:     0 d,  0 h,  0 min,  0.010 sec
 *  cpu-time:     0 d,  0 h,  0 min,  0.056 sec
 * ratio c/w:     5.698 speedup

